api/src/db/models.js

```diff
@@ -0,0 +1,34 @@
+const mongoose = require('mongoose');
+
+const schemas = {
+  wod: {
+    releaseDate: {
+      type: Date,
+      required: true,
+    },
+    videoUrl: {
+      type: String,
+      required: true,
+    },
+    rating: mongoose.Schema.Types.Decimal128,
+    type: {
+      type: String,
+      enum: ['FT', 'RFT', 'AMRAP', 'CHIPPER', 'STATIONS'],
```

Comment:
Could you add the value of EMOM, please?

Reply:
Yep, I missed it!